### PR TITLE
Added Makefile for running benchmarks.

### PR DESCRIPTION
**Pull request by james-perretta**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+benchmark_files := $(notdir $(wildcard benchmarks/*.scm))
+correct_files := $(benchmark_files:.scm=.correct)
+benchmark_targets := $(benchmark_files:.scm=.bench)
+
+all: $(benchmark_targets)
+
+%.bench:
+	@echo $@
+	@mkdir -p data
+	@(time ./scheme.py $(FLAGS) \
```
Comment (porglezomp) [inline Makefile]: Time is included in `-s` now, so doing `time` might not be necessary anymore here

Comment (james-perretta) [reply Makefile]: Ah ok

File: benchmarks/game_of_life.correct [hunk 2]
```diff
@@ -1,41 +1,41 @@
-5555555
-5001115
-5000005
-5100005
-5111115
-5101005
-5555555
+-------
```
Comment (porglezomp) [inline benchmarks/game_of_life.correct:1]: How much extra effort is it to use `+` for the corners? :P

Comment (james-perretta) [reply benchmarks/game_of_life.correct:1]: Too much : P

Comment (porglezomp) [reply benchmarks/game_of_life.correct]: wait not it's not!

File: benchmarks/game_of_life.scm [hunk 3]
```diff
@@ -127,9 +127,9 @@
 
 (define (print-grid-helper rows cols current-row grid)
   (if (<= current-row rows)
-      (begin (display 5)
+      (begin (display '|)
```
Comment (porglezomp) [inline benchmarks/game_of_life.scm]:
```suggestion
      (begin (display '+)
```

File: benchmarks/game_of_life.scm [hunk 4]
```diff
@@ -127,9 +127,9 @@
...
              (print-grid-row rows cols current-row 1 grid)
-             (display 5)
+             (display '|)
```
Comment (porglezomp) [inline benchmarks/game_of_life.scm]:
```suggestion
             (display '+)
```

**Review by porglezomp — APPROVED** [review]

**Pull request merged**